tests/x11/remote_desktop/onetime_vncsession_xvnc_tigervnc.pm

```diff
@@ -38,17 +38,15 @@ sub run {
 
     # Start vncviewer and login with fullscreen
     x11_start_program('vncviewer', target_match => 'vnc_password_dialog');
-    type_string '10.0.2.1:1';
+    type_string_slow '10.0.2.1:1';
```

Comment:
Can you say why we need to type slower here? And if it's really necessary then please give a reasoning in a comment above this line

Reply:
Sometimes it will execute the next line (assert_and_click...) even the type_string is not finished.
type_string_slow will improve the situation but I added a wait_still_screen after this line instead.